package.json

```diff
@@ -25,6 +25,7 @@
     "typeahead.js": "~0.10.5",
     "temp": "~0.7.0",
     "walk": "~2.3.3",
-    "d3": "3.5.5"
+    "d3": "3.5.5",
+    "fuse": "krisk/Fuse#master"
```

Comment:
I depend on the master as a used function only has been added after the latest release